lib/addons/fluxcd/index.ts

```diff
@@ -10,22 +10,9 @@ import { FluxKustomization } from "./kustomization";
 import { KubernetesManifest } from 'aws-cdk-lib/aws-eks/lib/k8s-manifest';
 
 /**
- * User provided options for the Helm Chart
+ * Options for a FluxCD repository
  */
-export interface FluxCDAddOnProps extends HelmAddOnUserProps {
-
-  /**
-   * Namespace where add-on will be deployed. 
-   * @default flux-system
-   */
-  namespace?: string;
-
-  /**
-  * Helm chart version to use to install.
-  * @default 2.8.0
-  */
-  version?: string;
-
+export interface FluxCDGitRepo {

```

Comment:
It should derive from `GitopsApplicationDeployment ` and add fields that are flux specific.